src/key.rs

```diff
@@ -127,6 +126,11 @@ impl Serialize for PrivateKey {
     }
 
     fn from_bytes(raw: &[u8]) -> Result<Self, Error> {
+        const FR_SIZE: usize = (Fr::NUM_BITS as usize + 8 - 1) / 8;
+        if raw.len() != FR_SIZE {
```

Comment:
Is it possible that `raw` is a stream of bytes containing more than one key (when `raw.len() % FR_SIZE == 0 && raw.len() > FR_SIZE`)? Is it possible that we call `PublicKey::from_btyes(raw)` on `raw` multiple times to read multiple sequential keys?

Reply:
if we do so, I never intended the api to be used like that